conductor_api/src/interface.rs

```diff
@@ -968,7 +971,11 @@ impl ConductorApiBuilder {
 }
 
 pub trait Interface {
```

Comment:
@lucksus or @maackle can we get a description of this trait here, since it could be important in the future, for implementing other interfaces?

It can be pulled roughly from the description of the PR I think